src/gemini_computer_use/main.py

```diff
@@ -2,6 +2,7 @@
 import re
 import sys
 import time
+import PIL.PngImagePlugin
```

Comment:
Is there a specific reason for this import?

Reply:
Need for windows.
https://github.com/googleapis/python-genai/pull/12